web/topogen.sh

```diff
@@ -46,7 +46,7 @@ echo "Extract metadata"
 echo 'Parsing countries..'
 ("$NODE_MODULES_PATH/shp2json" -n build/${COUNTRIES_FILENAME}.shp \
   | "$NODE_MODULES_PATH/ndjson-map" '(d.id = (d.properties.ISO_A3 != "-99") ? d.properties.ISO_A3 : d.properties.ADM0_A3, d.oldProps = d.properties, d.properties = {}, d.properties.subid = d.oldProps.SU_A3, d.properties.code_hasc = d.oldProps.code_hasc, delete d.oldProps, d)' \
-  | "$NODE_MODULES_PATH/ndjson-filter" "d.id && ${STATES_FILTER}.indexOf(d.id) === -1" \
```

Comment:
No longer needed?

Reply:
This will exclude countries that are in the states list. That's a harsh constraint. We might want to be able to access country geometries even though we split it by state.
Also we don't really need to filter anything at this stage anymore as we do not pass the whole list of possible geometries but instead keep only the ones we need.